src/test/java/de/azapps/kafkabackup/common/record/RecordJSONSerdeTest.java

```diff
@@ -2,120 +2,110 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.module.SimpleModule;
-import de.azapps.kafkabackup.common.record.Record;
 
 import static org.junit.Assert.assertEquals;
 
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.*;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.nio.file.Files;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
 import java.util.Base64;
-import java.util.Comparator;
 
 public class RecordJSONSerdeTest {
 
+    private static final String topic = "test-topic";
+    private static final int partition = 42;
+    private static final long offset = 123;
+    private static byte[] keyBytes;
+    private static String keyBase64;
+    private static byte[] valueBytes;
+    private static String valueBase64;
+    private static final String JSON_ENCODING = "UTF-8";
+    private static ObjectMapper mapper;
+    private static RecordJSONSerde serde;
+
+    @BeforeClass
+    public static void beforeAll() throws Exception {
+        // encoding here is not really important, we just want some bytes
+        keyBytes = "test-key".getBytes("UTF-8");
+        valueBytes = "test-value".getBytes("UTF-8");
+        // using Base64 as encoding in the json is part of our Record serialization format however:
+        keyBase64 = Base64.getEncoder().encodeToString(keyBytes);
+        valueBase64 = Base64.getEncoder().encodeToString(valueBytes);
+    }
+
+    @Before
```

Comment:
btw we can try to use JUnit 5(instead of JUnit 4) - besides some features, for example this annotation has better name in JUnit5 which is @BeforeEach - more readable

Reply:
Yes. Would love that. Will try!